examples/server/webui/src/main.js

```diff
@@ -55,7 +55,7 @@ const CONFIG_DEFAULT = {
   temperature: 0.8,
   dynatemp_range: 0.0,
   dynatemp_exponent: 1.0,
-  top_k: 40,
+  top_k: 1,
```

Comment:
Since this is not in sync with `common.h` anymore, I think we should add a comment too. Just in case in the future we may want to pull these default values from `/props` endpoint